package.json

```diff
@@ -109,6 +109,7 @@
     "write-file-webpack-plugin": "^4.2.0"
   },
   "dependencies": {
+    "vee-validate": "^2.0.1",
```

Comment:
Not really necessary for this, but we should pick versions and not use `^<version>`